src/dowel/csv_output.py

```diff
@@ -48,7 +51,25 @@ def record(self, data, prefix=''):
                            'Did you change key sets after your first '
                            'logger.log(TabularInput)?'.format(
                                set(self._fieldnames), set(to_csv.keys())))
-
+                with open(self._filename, "r") as old:
```

Comment:
The warning above this line can be deleted now, since this case is handled. Please also delete any private code that becomes unused by that change.

Reply:
No problem! Thank you for pointing out!